### PR TITLE
Add driver and example

### PR DESCRIPTION
**Pull request by peteGSX**
PR to add the PCA/TCA9555 device driver (including updating the myHal.cpp example) as contributed by Henkk via Discord.

### CODE REVIEW

**Comment by Asbelos** [comment]
Henkk should add his name to the copyright.

File: myHal.cpp_example.txt [hunk 1]
```diff
@@ -60,6 +63,33 @@ void halSetup() {
...
+  //PCA9555::create(196, 16, 0x22);
+
+  // Alternative form, which allows the INT pin of the module to request a scan
+  // by pulling Arduino pin 22 to ground. Means that the I2C isn't being polled
+  // all the time, only when a change takes place. Multiple modules' INT pins
+  // may be connected to the same Arduino pin.
+
+  //PCA9555::create(196, 16, 0x22, 22);
+  
+
```
Comment (JeaneLG) [inline myHal.cpp_example.txt:83]: The following lines (84-92) are a duplicate resulting from copy-and-paste.

**Review by JeaneLG — CHANGES_REQUESTED** [review]
please change the code regarding my comment on file 'myHal.cpp_example.txt'

**Comment by FrightRisk** [comment]
So this has been tested by 2 people? I don't have a board to test.

**Comment by peteGSX** [comment]
This has been added in a subsequent release.

**Pull request closed without merging**